test/js/components/tasks/submit.test.jsx

```diff
@@ -40,7 +41,7 @@ describe('<SubmitModal/>', () => {
     test('closes modal', () => {
       const { getByText, toggleModal } = setup();
 
-      expect(getByText('Submit this task for review')).toBeVisible();
+      expect(getByText('Submit Task for Review')).toBeVisible();
```

Comment:
I'm not clear why this changed.

Reply:
The tests failed with the original version and showed me the text they were looking at, and it had that slightly changed version, so I changed it, on the assumption that some user-facing text had changed. Perhaps I was wrong?

Reply:
I don't think we're _intentionally_ changing user-facing text for the submit-task modal in this PR, so if the test is failing I think that represents a bug we want to surface.